examples/confined_users-playbook.yml

```diff
@@ -0,0 +1,107 @@
+---
+- name: Manage SELinux policy example
+  hosts: all
+  become: true
+  vars:
+    # Use "targeted" SELinux policy type
+    selinux_policy: targeted
+    # Set "enforcing" mode
+    selinux_state: enforcing
+    # Set SELinux booleans
+    selinux_booleans:
+      # Allow ssh logins as sysadm_r:sysadm_t
+      - {name: 'ssh_sysadm_login', state: 'on', persistent: 'yes'}
+      # Allow sysadm_u/staff_u to execute applications in their
+      # home directories and /tmp
+      - {name: 'sysadm_exec_content', state: 'on', persistent: 'yes'}
+      - {name: 'staff_exec_content', state: 'on', persistent: 'yes'}
+      # But block normal users from the same
+      - {name: 'user_exec_content', state: 'off', persistent: 'yes'}
+      # Allow staff user to create and transition to svirt domains
+      #- {name: 'staff_use_svirt', state: 'on', persistent: 'yes'}
+      # Allow confined users the ability to execute the ping and traceroute commands
+      #- {name: 'selinuxuser_ping', state: 'on', persistent: 'yes'}
+      # Allow users to connect to the local mysql server
+      #- {name: 'selinuxuser_mysql_connect_enabled', state: 'on', persistent: 'yes'}
+      # Allow samba and winbind-rpcd to share users home directories
+      #- {name: 'samba_enable_home_dirs', state: 'on', persistent: 'yes'}
+      # Disallow programs, such as newrole, from transitioning to administrative user domains
+      #- {name: 'secure_mode', state: 'on', persistent: 'yes'}
+
+    # List of privileged users to be created
+    privileged_users:
+      - "staff"
+      - "roger"
+
+    # The following section specifies mapping of EXISTING system users to SELinux users
+    selinux_logins:
+      # Map all new users to SElinux user "user_u" (instead of "unconfined_u")
+      - {login: '__default__', seuser: 'user_u', serange: 's0', state: 'present'}
+      # Alternatively map all new users to SElinux user "staff_u" (instead of "unconfined_u")
+      #- {login: '__default__', seuser: 'staff_u', serange: 's0-s0:c0.c1023', state: 'present'}
+      # Map admin users to SElinux user "sysadm_u"
+      - {login: 'root', seuser: 'sysadm_u', serange: 's0-s0:c0.c1023',
+               state: 'present'}
```

Comment:
In "mls" we use "root" SELinux user for "root" user and `sysadm_u` for admin non-root users. I'd follow it here as well.

Reply:
Ok, I guess `root` can stay "unconfined" since it's not accessible from outside. I'll replace it with a better example.